### ISSUE TITLE
SourceFile.hs: Constructor ‘D.ImportDeclaration’ should have 4 arguments, but has been given 3

### ISSUE THREAD

**Issue by juhp** [kRITZCREEK/psc-ide#115]
While build Stackage Nightly I ran into:

```
[ 8 of 12] Compiling PureScript.Ide.SourceFile ( lib/PureScript/Ide/SourceFile.hs, dist/build/PureScript/Ide/SourceFile.o )

lib/PureScript/Ide/SourceFile.hs:43:25:
    Constructor ‘D.ImportDeclaration’ should have 4 arguments, but has been given 3
    In the pattern: D.ImportDeclaration mn importType' qualifier
    In an equation for ‘mkModuleImport’:
        mkModuleImport (D.ImportDeclaration mn importType' qualifier)
          = ModuleImport
              (T.pack (N.runModuleName mn))
              importType'
              (T.pack . N.runModuleName <$> qualifier)
    In an equation for ‘getImportsForFile’:
        getImportsForFile fp
          = do { module' <- parseModuleFromFile fp;
                 let imports = ...;
                 return
                   (fmap (mkModuleImport . unwrapPositionedImport) <$> imports) }
          where
              mkModuleImport (D.ImportDeclaration mn importType' qualifier)
                = ModuleImport
                    (T.pack (N.runModuleName mn))
                    importType'
                    (T.pack . N.runModuleName <$> qualifier)
              mkModuleImport _
                = error "Shouldn't have gotten anything but Imports here"
              unwrapPositionedImport
                (D.ImportDeclaration mn importType' qualifier)
                = D.ImportDeclaration mn (unwrapImportType importType') qualifier
              unwrapPositionedImport x = x
              unwrapImportType (D.Explicit decls)
                = D.Explicit (map unwrapPositionedRef decls)
              unwrapImportType (D.Hiding decls)
                = D.Hiding (map unwrapPositionedRef decls)
              unwrapImportType D.Implicit = D.Implicit

lib/PureScript/Ide/SourceFile.hs:49:33:
    Constructor ‘D.ImportDeclaration’ should have 4 arguments, but has been given 3
    In the pattern: D.ImportDeclaration mn importType' qualifier
    In an equation for ‘unwrapPositionedImport’:
        unwrapPositionedImport
          (D.ImportDeclaration mn importType' qualifier)
          = D.ImportDeclaration mn (unwrapImportType importType') qualifier
    In an equation for ‘getImportsForFile’:
        getImportsForFile fp
          = do { module' <- parseModuleFromFile fp;   
                 let imports = ...;
                 return
                   (fmap (mkModuleImport . unwrapPositionedImport) <$> imports) }
          where
              mkModuleImport (D.ImportDeclaration mn importType' qualifier)
                = ModuleImport
                    (T.pack (N.runModuleName mn))
                    importType'
                    (T.pack . N.runModuleName <$> qualifier)
              mkModuleImport _
                = error "Shouldn't have gotten anything but Imports here"
              unwrapPositionedImport
                (D.ImportDeclaration mn importType' qualifier)
                = D.ImportDeclaration mn (unwrapImportType importType') qualifier
              unwrapPositionedImport x = x
              unwrapImportType (D.Explicit decls)
                = D.Explicit (map unwrapPositionedRef decls)
              unwrapImportType (D.Hiding decls)
                = D.Hiding (map unwrapPositionedRef decls)
              unwrapImportType D.Implicit = D.Implicit
```

Not sure why this happened now.

**Comment by kRITZCREEK** [comment]
Should be fixed with 0.6.0 release. Could you give it another spin?

**Comment by juhp** [comment]
Thanks

cc @snoyberg (this week's stackage curator)
